src/API/Google/Query/AdsAssetGroupAssetQuery.php

```diff
@@ -0,0 +1,22 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Query;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class AdsAssetGroupAssetQuery
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Query
+ */
+class AdsAssetGroupAssetQuery extends AdsQuery {
+
+	/**
+	 * Query constructor.
+	 */
+	public function __construct() {
```

Comment:
💅  Would be cool to ad some ref about the parameters available here in case another dev needs to update this query.

Reply:
Do you mean the methods and properties available from the `AdsQuery` and `Query` class?

Reply:
I was thinking about a link where you found the documentation for the available columns and params in the query.